pageserver/Cargo.toml

```diff
@@ -32,11 +32,13 @@ tokio-postgres = { git = "https://github.com/zenithdb/rust-postgres.git", rev="a
 postgres-types = { git = "https://github.com/zenithdb/rust-postgres.git", rev="a0d067b66447951d1276a53fb09886539c3fa094" }
 postgres-protocol = { git = "https://github.com/zenithdb/rust-postgres.git", rev="a0d067b66447951d1276a53fb09886539c3fa094" }
 postgres = { git = "https://github.com/zenithdb/rust-postgres.git", rev="a0d067b66447951d1276a53fb09886539c3fa094" }
+rocksdb = { git = "https://github.com/rust-rocksdb/rust-rocksdb.git" }
```

Comment:
Why are we picking the upstream development branch? Do we need features that don't exist in any of the released versions?

If we need the bleeding-edge code, Please pin to a particular git rev (see the lines right above it). Otherwise, every time we build we may be getting a different version of the rocksdb code.